client/src/main/java/org/apache/rocketmq/client/impl/consumer/RebalanceImpl.java

```diff
@@ -208,21 +208,16 @@ public void lockAll() {
                     Set<MessageQueue> lockOKMQSet =
                         this.mQClientFactory.getMQClientAPIImpl().lockBatchMQ(findBrokerResult.getBrokerAddr(), requestBody, 1000);
 
-                    for (MessageQueue mq : lockOKMQSet) {
+                    for(MessageQueue mq : mqs){
```

Comment:
Please re-format your codes

Reply:
ok thanks, now the code is formatted，you can review it again